app/src/androidTest/java/com/duckduckgo/app/analyticsSurrogates/AnalyticsSurrogatesLoaderTest.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2018 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.analyticsSurrogates
+
+import android.support.test.InstrumentationRegistry
+import com.duckduckgo.app.analyticsSurrogates.store.AnalyticsSurrogatesDataStore
+import org.junit.Assert
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class AnalyticsSurrogatesLoaderTest {
```

Comment:
Likewise, I think should drop Analytics in any of the names through out. (I won't flag any others)

Reply:
Will replace `AnalyticsSurrogate` with `ResourceSurrogate` throughout